docfx.json

```diff
@@ -0,0 +1,48 @@
+{
+  "metadata": [
+    {
+      "src": [
+        {
+          "src": "./src",
+          "files": [
+            "**/*.csproj",
+            "**/bin/**/**LaunchDarkly**.dll"
+          ]
+        }
+      ],
+      "dest": "./api",
+      "properties" : {
+        "Configuration": "Debug"
+      }
+    }
+  ],
+  "build": {
+    "content": [
+      {
+        "files": [
+          "**/*.{md,yml}"
+        ],
+        "exclude": [
+          "docs/**"
+        ]
+      }
+    ],
+    "resource": [
+      {
+        "files": [
+          "images/**"
+        ]
+      }
+    ],
+    "output": "docs",
+    "template": [
+      "default"
+    ],
+    "globalMetadata": {
+      "_appName": "LaunchDarkly Logging API for .NET - Common.Logging Adapter",
```

Comment:
Should it be named something different

Reply:
Yes, I missed the logging config and snk and will update those.